client/src/components/auth/authComponents/LoginForm.jsx

```diff
@@ -1,79 +1,94 @@
 /**
  * This file exports an Auth component used to display sign-ins and sign-ups.
  */
-import React, { useState } from 'react';
+import React from 'react';
 import { Jumbotron, Form } from 'react-bootstrap';
 import '../css/login-form.css';
 import { useDispatch } from 'react-redux';
 import { Button } from '@material-ui/core';
+import { Formik } from 'formik';
+import * as yup from 'yup';
 import { login } from '../../../actions/auth';
 
 const LoginForm = () => {
   const dispatch = useDispatch();
-  const [inputs, setInputs] = useState({
-    email: '',
-    password: '',
+
+  const teamInfoSchema = yup.object({
+    email: yup
+      .string()
+      .required('Please Enter your Email'),
```

Comment:
Can we have only have the first word with the first uppercase? Eg. 'Please enter your email'